Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/core/model/BaseCoronaTest.kt

```diff
@@ -0,0 +1,164 @@
+package de.rki.coronawarnapp.familytest.core.model
+
+import com.google.gson.annotations.SerializedName
+import de.rki.coronawarnapp.coronatest.server.CoronaTestResult
+import de.rki.coronawarnapp.coronatest.type.CoronaTest
+import de.rki.coronawarnapp.coronatest.type.CoronaTestDcc
+import de.rki.coronawarnapp.coronatest.type.CoronaTestUiState
+import de.rki.coronawarnapp.coronatest.type.RegistrationToken
+import de.rki.coronawarnapp.coronatest.type.TestIdentifier
+import org.joda.time.Instant
+import org.joda.time.LocalDate
+
+data class BaseCoronaTest(
```

Comment:
What about switching , we call the model `CoronaTest` and the interface `BaseCoronaTest`?
I think this could be confusing, specially that we can not extend `data` classes in Kotlin

Reply:
I switched, which added plenty of changes, but it's only renaming